lib/simplecov/version.rb

```diff
@@ -1,5 +1,5 @@
 module SimpleCov
-  version = "0.14.1"
+  version = "0.14.1".dup
 
   def version.to_a
     split(".").map(&:to_i)
```

Comment:
wait, whaaaaaat? I think I'd rather kill this code than introducing the `.dup` :see_no_evil: